extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -139,8 +151,10 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 			return true;
 		}).component();
 		this._fileShareSubscriptionDropdown.onValueChanged(async (value) => {
-			this._fileShare.subscriptionId = (this._fileShareSubscriptionDropdown.value as azdata.CategoryValue).name;
-			await this.loadFileShareStorageDropdown();
+			if (this._fileShareSubscriptionDropdown.value) {
```

Comment:
Why are you fetching the value from the object instead of using the value parameter?

Reply:
The value parameter only contains the displayName and not the entire azdata.CategoryValue.

Reply:
That doesn't seem right - so we're expecting people to do the lookup themselves if they want to do something with the full category value? If the user passed in category values originally then onValueChanged should be sending those as the value too I would expect.

Reply:
Talked to Aasim offline about this - onValueChanged is currently sending only the display name in the value object. This isn't great behavior but to change it would be a breaking change for anyone relying on that. So while we should fix it at some point for the purposes of this PR this is fine.